ledger/src/blockstore.rs

```diff
@@ -3031,6 +3034,7 @@ impl Blockstore {
         slot: u64,
         index: u32,
         new_shred_raw: &[u8],
+        // TODO: change arg type to ShredType.
```

Comment:
Want to change this now? I don't see too many callers.

Reply:
done